ee/localserver/server.go

```diff
@@ -415,13 +416,14 @@ func (ls *localServer) presenceDetectionHandler(next http.Handler) http.Handler
 
 		// presence detection is only supported on macos currently
 		if runtime.GOOS != "darwin" {
+			w.Header().Add(kolideDurationSinceLastPresenceDetectionHeaderKey, presencedetection.DetectionFailedDurationValue.String())
```

Comment:
Thinking about this, we'll need to figure out how to differentiate between "not supported" and "timeout" or "error"

Reply:
I adjusted it so we only return the presence response header when presence detection is actually requested.

That might give us enough data for some logic like

```
if failure && os == "linux"
  // we no it's not supported
```

but maybe we need something more concrete like a failure_reason header